artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: etcdproxy-clusterrole
+rules:
+- apiGroups: ["extensions", "apps"]
```

Comment:
just apps I'd think.  No reason for you to use a deprecated API

Reply:
Removed deployment rule in favor of ReplicaSets rule.